sdk/core/core-client-rest/src/urlHelpers.ts

```diff
@@ -24,6 +24,22 @@ export function buildRequestUrl(
     return path;
   }
 
+  if (options.pathParameters) {
+    const pathParams = options.pathParameters;
+    for (const key of Object.keys(pathParams)) {
+      const param = pathParams[key] as any;
+      if (param === undefined || param === null) {
```

Comment:
I think we may need to handle undefined and null differently here. If we skip them for path parameters, we'll end up with an invalid URL containing `{param}`. So we may need to throw here and also make sure that the customURL params are marked as required when generating the RLC types